src/clients/BundleDataClient/utils/SuperstructUtils.ts

```diff
@@ -85,12 +85,13 @@ const V3RelayExecutionEventInfoSS = object({
   updatedOutputAmount: BigNumberType,
   fillType: FillTypeSS,
   updatedRecipient: string(),
-  updatedMessage: string(),
+  updatedMessage: optional(string()),
```

Comment:
I thought we didn't want  optional types?

Reply:
In this case I think optional() is preferred because it allows for the presence of message, but doesn't auto-populate it in the retrieved data. After 7 days we'll be OK to remove this.